docs/source/contributing/contributing.md

```diff
@@ -1,45 +1,86 @@
 # Contributing to repo2docker development
 
+Thank you for thinking about contributing to repo2docker!
+This is an open source project that is developed and maintained entirely by volunteers.
+*Your contribution* is integral to the future of the project.
+THANK YOU!
+
+## Types of contribution
+
+There are many ways to contribute to repo2docker:
+
+* **Update the documentation.**
+  If you're reading a page or docstring and it doesn't make sense (or doesn't exist!), please let us know by opening a bug report.
+  It's even more amazing if you can give us a suggested change.
+* **Fix bugs or add requested features.**
+  Have a look through the [issue tracker](https://github.com/jupyter/repo2docker/issues) and see if there are any tagged as ["help wanted"](https://github.com/jupyter/repo2docker/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+  As the label suggests, we'd love your help!
+* **Report a bug.**
+  If repo2docker isn't doing what you thought it would do then open a [bug report](https://github.com/jupyter/repo2docker/issues/new?template=bug_report.md).
+  That issue template will ask you a few questions described in more detail below.
+* **Suggest a new feature.**
+  We know that there are lots of ways to extend repo2docker!
+  If you're interested in adding a feature then please open a [feature request](https://github.com/jupyter/repo2docker/issues/new?template=feature_request.md).
+  That issue template will ask you a few questions described in detail below.
+* **Tell people about repo2docker.**
```

Comment:
```suggestion
* **Review someone's Pull Request.**
  Whenever somebody proposes changes to the repo2docker codebase, the community reviews
  the changes, and provides feedback, edits, and suggestions. Check out the
  [open pull requests](https://github.com/jupyter/repo2docker/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc)
  and provide feedback that helps improve the PR and get it merged. Please keep your
  feedback positive and constructive!
```

A last second thought - what do you think about this?

Reply:
Yeah! Good call!!

What are your formatting guides for markdown files? I like one sentence per line (which is what I’ve done for the other sentences I think) but I can change that if you’d like?

Reply:
I don't think we have strict guidelines for it, I usually just try to add linebreaks in a way that's visually-appealing assuming there's no wrapping going on